README.md

```diff
@@ -22,22 +22,15 @@ deploy and use the latest release, please see the
 ## Prerequisites
 
 You must have [Docker Engine](https://docs.docker.com/install/) installed to
-build the Splunk Operator.
+build the Splunk Operator. 
 
 This project uses [Go modules](https://blog.golang.org/using-go-modules),
 and requires [golang](https://golang.org/doc/install) 1.17.3 or later.
 You must `export GO111MODULE=on` if cloning these repositories into your
 `$GOPATH` (not recommended).
 
 The [Kubernetes Operator SDK](https://github.com/operator-framework/operator-sdk)
-must also be installed to build this project.
-
-```shell
-git clone -b v1.15.0 https://github.com/operator-framework/operator-sdk
-cd operator-sdk
-make tidy
-make install
-```
+must also be installed to build this project. Follow the steps from [Operator-SDK Installation Documentaion](https://sdk.operatorframework.io/docs/installation/)
```

Comment:
With Operator SDK version regularly updating, customer/admin might end up using a later version. I think we should just roll back to our previous specific guidance of installing 1.15 version

Reply:
this is official way of installing operator-sdk. they always update this document.

Reply:
Ok, will update the document

Reply:
fixed